source/documentation/airflow.md

```diff
@@ -333,10 +346,75 @@ To deploy your Airflow sandbox, follow the instructions in the [Work with Analyt
 
 Deploying your Airflow sandbox will create an `airflow` folder in your home directory on the Analytical Platform. This folder contains three subfolders: `db`, `dags` and `logs`.
 
-Once you have deployed your Airflow sandbox, you should store the script for the DAG you want to test in the `airflow/dags` folder in your home directory on the Analytical Platform. Airflow scans this folder every three minutes and will automatically detect your pipeline.
+Once you have deployed your Airflow sandbox, you should store the script for the DAG you want to test in the `airflow/dags` folder in your home directory on the Analytical Platform. Airflow scans this folder every three minutes and will automatically detect your DAG.
 
-To ensure that your pipeline runs correctly, you should set the `ROLE` variable in your DAG script to be your own IAM role on the Analytical Platform. This is your GitHub username in lowercase prefixed with `alpha_user_`. For example, if your GitHub username was `Octocat-MoJ`, your IAM role would be `alpha_user_octocat-moj`.
+To ensure that your DAG runs correctly, you should set the `ROLE` variable in your DAG script to be your own IAM role on the Analytical Platform. This is your GitHub username in lowercase prefixed with `alpha_user_`. For example, if your GitHub username was `Octocat-MoJ`, your IAM role would be `alpha_user_octocat-moj`.
 
 You should also set the `NAMESPACE` variable in your DAG script to be your own namespace on the Analytical Platform. This is your GitHub username in lowercase prefixed with `user-`. For example, if your GitHub username was `Octocat-MoJ`, your namespace would be `user-octocat-moj`.
 
 You should only use your Airflow sandbox for testing purposes.
+
+## Troubleshooting
+
+If a task fails in Airflow, you can click on it and view the logs. The final few lines before `Event with job id your-task-id-9stuff876 Failed` should indicate the problem. As always, it is a good idea to have print statements in your code to help identify the stage at which it failed.
+Here are some common errors you might encounter:
+
+- **DAG doesn't start on schedule** Before initial use, a DAG needs to be switched on using the Airflow UI (toggle in top left). If you used the homepage toggles, please be very careful not to turn off other critical DAGs. The Airflow UI can misbehave a lot and sometimes the homepage toggles are just wrong, so better to use the toggle within a DAG menu.
```

Comment:
I think these might be better as sections with subheadings as opposed to a bullet list.